reactioncommerce/reaction
# feat: add shopLogoUrls to shop settings to allow logo to be rendered from url

Impact: **minor**  
Type: **feature**

Adds a field to allow operators to link to an existing logo URL, and use that logo / url as their primary shop logo.

## Breaking changes
None. The URL logo will take precedence, if available, but will not affect anything if it is not added.

## Testing
1. Start the app, see that the default reaction logo is used as the shop logo.
1. Upload a logo via the upload image feature, and see that the uploaded image takes over as the shop logo.
1. Add an image URL,  and see that the image form the URL takes over as the shop logo.
1. Remove the URL, and see the image goes back to the uploaded image.
1. Remove uploaded image, and see it goes back to the default
1. Add a URL (without uploading an image), and see that the URL logo becomes the shop logo.
